### PR TITLE
Update locator.ts

### PR DESCRIPTION
**Pull request by rcwd**
"Implemented a mechanism to prevent lazy loading for IMG or DIV elements with the CSS class .nolazy.

### CODE REVIEW

**Comment by erikyo** [comment]
Ciao @rcwd, 

First of all, thank you for the pull request. I am pleased to see contributors helping me improve the modules I created!

Regarding your proposal, it looks good. However, I realize I mistakenly did not direct you to the correct part of the code for this modification. The right place to make this change is when we are adding the mutation observer for new elements. If an element has the 'no-lazy' class, we can immediately skip processing it, thus avoiding the steps of checking if the element is in the viewport and then discarding it.

In short, could you please update the pull request to eagerly skip elements with the 'no-lazy' class?

I believe the appropriate place for this change is within the "if" statement here:
https://github.com/erikyo/auto-lazyload/blob/889d50f413f5739479d5e135edfc53ffbbf357ac/src/autoLazy.ts#L21-L31

and the code required should be like this:
```
if (isElement.classList.contains('no-lazy')) {
    continue;
}
```

(feel free to change or update my code, it's untested and it's just for the purpose of showing what I mean)
